docs/core/tools/dotnet-new-update.md

```diff
@@ -9,49 +9,81 @@ ms.date: 04/29/2021
 
 ## Name
 
-`dotnet new --update-check` - Checks for available updates for installed template packages.
-
-`dotnet new --update-apply` - Applies updates to installed template packages.
+`dotnet new update` - updates installed template packages.
+`dotnet new update --check-only` - Checks for available updates for installed template packages without applying them.
 
 ## Synopsis
 
 ```dotnetcli
-dotnet new --update-check
+dotnet new update [--interactive] [--add-source|--nuget-source <SOURCE>] 
+    [-d|--diagnostics] [--verbosity <LEVEL>] [-h|--help]
 
-dotnet new --update-apply
+dotnet new update --check-only|--dry-run [--interactive] [--add-source|--nuget-source <SOURCE>] 
+    [-d|--diagnostics] [--verbosity <LEVEL>] [-h|--help]
 ```
 
 ## Description
 
-The `dotnet new --update-check` option checks if there are updates available for the template packages that are currently installed.
-The `dotnet new --update-apply` option checks if there are updates available for the template packages that are currently installed and installs them.
+The `dotnet new update` command updates installed template packages.
+The `dotnet new update` command with `--check-only` option checks for available updates for installed template packages without applying them.
 
 > [!NOTE]
 > [!INCLUDE [new syntax](../../../includes/dotnet-new-7-0-syntax.md)]
 >
-> Examples of the new syntax:
+> Examples of the old syntax:
 >
 > - Show help for the `update` subcommand.
 >
->   ```dotnetcli
->   dotnet new update --help
->   ```
->
 > - Check for updates for installed template packages:
 >
 >   ```dotnetcli
->   dotnet new update --check
+>   dotnet new --update-check
 >   ```
 >
 > - Update installed template packages:
 >
 >   ```dotnetcli
->   dotnet new update
+>   dotnet new --update-apply
 >   ```
 
+## Options
+
+[!INCLUDE [interactive](../../../includes/cli-interactive-5-0.md)]
+
+- **`--add-source|nuget-source <SOURCE>`**
+  
+  By default, `dotnet new install` uses the hierarchy of NuGet configuration files from the current directory to determine the NuGet source the package can be installed from. If `--nuget-source` is specified, the source will be added to the list of sources to be checked.  
+  To check the configured sources for the current directory use [`dotnet nuget list source`](dotnet-nuget-list-source.md). For more information, see [Common NuGet Configurations](/nuget/consume-packages/configuring-nuget-behavior). Available since .NET SDK 7.0.100.
+
+- **`-h|--help`**
+
+  Prints out help for the update command. Available since .NET SDK 7.0.100.
+
+- **`-d|--diagnostics`**
+
+  Enables diagnostic output. Available since .NET SDK 7.0.100.
+
```

Comment:
```suggestion
- **`-d|--diagnostics`**

  Enables diagnostic output. Available since .NET SDK 7.0.100.

- **`-h|--help`**

  Prints out help for the uninstall command. Available since .NET SDK 7.0.100.

```